data/seeds/012_Points.js

```diff
@@ -1,9 +1,21 @@
-const points = [...new Array(8)].map((i, idx) => ({
-  WritingPoints: 30,
-  DrawingPoints: 20,
-  MemberID: `${Math.floor((idx + 2) / 2)}`,
-  SubmissionID: `${idx + 1}`,
-}));
+const points = [
```

Comment:
These changes will absolutely expedite the testing process, great job Emilio!

Reply:
Thank you George! 😃